README.md

```diff
@@ -89,7 +74,7 @@ migration projects, which are always painful and complicated. In reality, the in
 relatively small, but today it is not possible for individual teams to handle their own storage needs.
 The Nexus provides the abstraction over the resources such that the developer teams stay in control.
 
-The reason we think this can work is because applications have changed, and the way they are built allows us to rethink
+The reason we think this can work is that applications have changed, and the way they are built allows us to rethink
```

Comment:
The statement is not articulated well, I think we can remove.

Reply:
You mean revert back to what it was or completely remove the line? Btw the change from because to that is suggested by the AI tool.